src/NotificationServices/WhatsAppMessageBirdNotification.php

```diff
@@ -85,12 +85,12 @@ public function via($notifiable)
     public function toWhatsApp($notifiable)
     {
         try {
-            if (is_null($this->imageUrl)) {
+            if (is_null($this->mediaUrl)) {
                 $response = WhatsAppMessageBird::sendWhatsAppText($this->to, $this->message, $this->reportUrl);
                 return $this->handleMessageSentEvent($response);
             }
     
-            $response = WhatsAppMessageBird::sendWhatsAppImage($this->to, $this->imageUrl, $this->message, $this->reportUrl);
+            $response = WhatsAppMessageBird::sendWhatsAppImage($this->to, $this->mediaUrl, $this->message, $this->reportUrl);
```

Comment:
Can you please rename sendWhatsAppImage's argument name to mediaUrl as well?

Reply:
updated at commit 80bb1a714df8346cd50375ebdadc31bca64f922e